department-of-veterans-affairs/va.gov-cms
# VACMS-12970: Add endpoint that outputs list of system menus for frontend consumption at build time.

## Description
See https://github.com/department-of-veterans-affairs/va.gov-team/issues/12970

## Testing done
Visual

## QA
As an admin, go to `/graphql/explorer` and enter the following query: ```{
  sideNavMenus
}```
This is what you should see:
![image](https://user-images.githubusercontent.com/2404547/95512596-0c81cc00-0987-11eb-900b-8200202209a4.png)
